README.md

```diff
@@ -1,3 +1,23 @@
 # contenta angular
 
-Coming soon!
+## Config the contenta URL inside the folder src/api/config.ts
+
+```
+export const ANGULAR_APP_JSONAPI = 'http://localhost/contenta/CONTENTACMS/web';
```

Comment:
Will ask about this in the chat. I didnt understand this was something we where aiming for..the app ofc can run on any server..but it needs to run node. because it needs to be able to render the app as it uses server rendering